src/routes/Player/OptionsMenu/OptionsMenu.js

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2017-2022 Smart code 203358507
+
+const React = require('react');
+const PropTypes = require('prop-types');
+const classnames = require('classnames');
+const Icon = require('@stremio/stremio-icons/dom');
+const { Button, useStreamingServer, useToast } = require('stremio/common');
+// const { useServices } = require('stremio/services');
+const styles = require('./styles');
+
+const OptionsMenu = ({ className, stream }) => {
+    const streamingServer = useStreamingServer();
+    // const { core } = useServices();
+    const toast = useToast();
+    const streamUrl = React.useMemo(() => {
+        return stream !== null ?
```

Comment:
ytId case must also be handled here.
we kind of handle it already in [external_player](https://github.com/Stremio/stremio-core/blob/development/src/deep_links/mod.rs#L239) but the url case is not suitable for this feature.
I suggest we refactor stremio-core by adding one more field `url` to ExternalPlayerLink which will be the same as href except for [this](https://github.com/Stremio/stremio-core/blob/development/src/deep_links/mod.rs#L31) case, then use deepLinks api here